modules/installation-requirements-user-infra.adoc

```diff
@@ -80,6 +102,65 @@ Each cluster machine must meet the following minimum requirements:
 
 |===
 
+ifeval::["{context}" == "installing-ibm-z"]
+[id="minimum-ibm-z-system-requirements_{context}"]
+== Minimum IBM Z system requirements
+
+=== Hardware requirements
+
+* 1 LPAR with 3 IFLs that supports SMT2
+* 1 OSA or RoCE network adapter, or both
+
+=== Operating system requirements
+
+* One instance of z/VM 7.1
+
+On your z/VM instance, set up:
+
+* 3 guest virtual machines for OCP control plane machines (master nodes)
```

Comment:
s/OCP/{product-title}
Avoid using "master nodes" and "worker nodes." The install team's expressed a preference for the "control plane machine" and "compute machine" nomenclature.

Reply:
Fixed